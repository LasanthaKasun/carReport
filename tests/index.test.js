import "@testing-library/jest-dom";
import renderer from 'react-test-renderer';
import Home from "../src/pages";

it("should render landing page", () => {
    const component = renderer.create(<Home />);
    expect(component).toMatchSnapshot()
});
