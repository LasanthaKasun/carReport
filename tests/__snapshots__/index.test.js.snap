// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render landing page 1`] = `
<div>
  <nav
    className="fixed z-30 w-full bg-white border-b-2 border-indigo-600"
  >
    <div
      className="px-6 py-3"
    >
      <div
        className="flex items-center justify-between"
      >
        <div
          className="flex items-center justify-start"
        >
          <a
            className="flex items-center text-xl font-bold"
            href="/"
          >
            <span
              className="text-blue-800"
            >
              Fixico
            </span>
          </a>
        </div>
        <div
          className="flex items-center"
        >
          <div
            className="relative inline-block "
          >
            <button
              className="relative flex items-center p-2 text-sm text-gray-600 bg-white border border-transparent rounded-md focus:border-blue-500 focus:ring-opacity-40 dark:focus:ring-opacity-40 focus:ring-blue-300 dark:focus:ring-blue-400 focus:ring dark:text-white dark:bg-gray-800 focus:outline-none"
            >
              <span
                className="mx-1"
              >
                Hi, Lasantha
              </span>
            </button>
            <div
              className="absolute right-0 z-20 w-56 mt-2 overflow-hidden bg-white rounded-md"
            />
          </div>
        </div>
      </div>
    </div>
  </nav>
  <div
    className="pt-12 lg:flex"
  >
    <div
      className="flex flex-col w-full px-4 py-8 overflow-y-auto border-b lg:border-r lg:h-screen lg:w-64"
    >
      <div
        className="flex flex-col justify-between mt-6"
      >
        <aside>
          <ul>
            <li>
              <a
                className="flex items-center px-4 py-2 text-gray-700 bg-gray-100 hover:bg-gray-200 rounded-md "
                href="/create"
              >
                <span
                  className="mx-1 font-medium"
                >
                  Damage Report
                </span>
              </a>
            </li>
            <li>
              <a
                className="flex items-center px-4 py-2 mt-5 text-gray-600 rounded-md bg-gray-100 hover:bg-gray-200"
                href="/list"
              >
                <span
                  className="mx-1 font-medium"
                >
                  View Overview
                </span>
              </a>
            </li>
          </ul>
        </aside>
      </div>
    </div>
    <div
      className="w-full h-full p-4 m-8 overflow-y-auto"
    >
      <div
        className="flex items-center justify-center p-16 mr-8 text-5xl font-bold"
        data-testid="message"
      >
        Welcome to the Fixico 2022
      </div>
    </div>
  </div>
</div>
`;
